src/ifcgeom/mapping/IfcCurveSegment.cpp

```diff
@@ -185,21 +170,26 @@ class curve_segment_evaluator {
 		auto Cy = C->as<IfcSchema::IfcCartesianPoint>()->Coordinates()[1];
 
 		eval_ = [L, Cx, Cy, theta, signX, fnX, signY, fnY](double u) {
+			using boost::math::quadrature::trapezoidal;
+
 			// integration limits, integrate from a to b
 			// from 8.9.3.19.1, integration limits are 0.0 to u where u is a normalized parameter
 			auto a = 0.0;
 			auto b = fabs(u / L);
 
-			auto n = 10; // use 10 steps in the numeric integration
+			// @todo where to plug this in?
```

Comment:
I don't think so. the docs say:

>  Since the routine is adaptive, step sizes are halved continuously until a tolerance is reached. In order to control this tolerance, simply call the routine with an additional argument

https://live.boost.org/doc/libs/1_68_0/libs/math/doc/html/math_toolkit/trapezoidal.html

So it's not just a fixed increment, but some limit to the number of refinement iterations. So conceptually it'd be more like 2^10 (big question mark). That's why I was a bit afraid to plug it in, but the default is 12. So I think we can just keep the defaults.